crates/core/src/contracts/tests/mod.rs

```diff
@@ -1,26 +1,2 @@
-use reth_primitives::U256;
```

Comment:
Be sure to use `git mv` next time, makes it easier to review.

Reply:
Ah that's weird, I did `git mv` for this [commit](https://github.com/kkrt-labs/kakarot-rpc/pull/546/commits/ff1093207b7898f272c668588ff6f7efd6854020) but somehow the aggregate changes doesn't display it because I also modify `mod.rs` afterwards 🤔